src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -745,11 +745,30 @@ private function preBoot(): ContainerInterface
         $container = $this->container;
 
         if ($container->hasParameter('kernel.trusted_hosts') && $trustedHosts = $container->getParameter('kernel.trusted_hosts')) {
-            Request::setTrustedHosts($trustedHosts);
+            Request::setTrustedHosts(\is_array($trustedHosts) ? $trustedHosts : preg_split('/\s*+,\s*+(?![^{]*})/', $trustedHosts));
```

Comment:
shouldn't this check that the parameter is a string before using `preg_split` on it ? Especially during booting, we need the code to be robust against unexpected parameter values (an old cache with an unsupported parameter might prevent you from running `cache:clear` in the prod environment)

Reply:
we don't do it for the other parameters so I didn't bother